docs/conf/scalar-prometheus-custom-values.yaml

```diff
@@ -34,6 +39,25 @@ grafana:
       # -- https://grafana.com/docs/grafana/latest/administration/configuration/#allow_embedding
       allow_embedding: false
 
+  # -- Additional data source configurations
+  additionalDataSources:
+  - name: Prometheus
+    type: prometheus
+    uid: prometheus
+    url: http://scalar-monitoring-kube-pro-prometheus.monitoring.svc.cluster.local:9090/
+    access: proxy
+    editable: false
+    isDefault: false
+    jsonData:
+      timeInterval: 30s
+  # - name: Loki
+  #   type: loki
+  #   uid: loki
+  #   url: http://scalar-logging-loki.monitoring.svc.cluster.local:3100/
+  #   access: proxy
+  #   editable: false
+  #   isDefault: false
```

Comment:
In the sample file, I commented out the Loki data source configuration by default since it is an optional thing.